build.gradle.kts

```diff
@@ -24,15 +24,15 @@ repositories {
 
 dependencies {
   implementation("com.atlassian.commonmark:commonmark:0.17.0")
-  implementation("com.google.code.gson:gson:2.8.8")
+  implementation("com.google.code.gson:gson:2.8.9")
   implementation("com.segment.analytics.java:analytics:3.1.3")
-  implementation("io.sentry:sentry:5.2.4")
+  implementation("io.sentry:sentry:5.3.0")
   implementation("io.snyk.code.sdk:snyk-code-client:2.1.12")
-  implementation("ly.iterative.itly:plugin-iteratively:1.2.10")
-  implementation("ly.iterative.itly:plugin-schema-validator:1.2.10") {
+  implementation("ly.iterative.itly:plugin-iteratively:1.2.11")
+  implementation("ly.iterative.itly:plugin-schema-validator:1.2.11") {
     exclude(group = "org.slf4j")
   }
-  implementation("ly.iterative.itly:sdk-jvm:1.2.10")
+  implementation("ly.iterative.itly:sdk-jvm:1.2.11")
```

Comment:
What's the reason to update versions above? 
Are we sure we don't get new bugs with such update?

Reply:
Yeah, it's only a minor upgrade and I tested it also manually against amplitude and verified events are sent.